### DISCUSSION TITLE
Is there a problem with the release process ?

### DISCUSSION THREAD

**Discussion by yvalette** [nektos/act#2343] [General]
At the time this is written, the last release is v0.2.62 and is at 17 commits to master.

I am struggling with an issue resolved in this commit https://github.com/nektos/act/commit/e1e5671e3d9ed4672fb58e90e0d9d064a0e45d9d, but as it is not released I cannot take advantage of it.

I am wondering about the release process and when the commit will be released.

**Comment by cplee** [comment]
Released on 5/31 with v0.2.63
